backend/main.go

```diff
@@ -15,8 +14,8 @@ func main() {
 	e := echo.New()
 
 	// Middleware
-	e.Use(middleware.Logger())
-	e.Use(middleware.Recover())
+				e.Use(middleware.Logger())
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
File is not `goimports`-ed